user-story/src/components/organims/Popup/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { Grid, TextField } from '@mui/material'
+import React from 'react'
+import Buttons from '../../atoms/buttons';
+import Icon from '../../atoms/icons';
+import TextTypography from '../../atoms/typography/index';
+
+interface PopupProps {
+    style?: React.CSSProperties;
+}
+
+const Popup = (props: PopupProps) => {
+    const [isEmailAdded ,setEmailAdded] = React.useState<boolean>(false);
+
+    const handleEmail = () => {
+        setEmailAdded(true);
+    };
+
+    const handleNeed = () => {
+
+    };
+  return (
+    <div>
+        <Grid container direction={"column"}
+         style={props.style}
+        >
```

Comment:
The popup itself is not a grid, it's a modal or dialog from mui, try to use that here